generated_code/acoustic.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+##
+# @file
+# This file is part of SeisSol.
+#
+# @author Carsten Uphoff (c.uphoff AT tum.de, http://www5.in.tum.de/wiki/index.php/Carsten_Uphoff,_M.Sc.)
```

Comment:
@JinwenPan, you are the author of this file. Add your name along with Carsten's in case you used his code and made modifications.